src/lkwdwrd/tests/phpunit/tmp/mu-plugins/zzz-mu-require.php

```diff
@@ -0,0 +1,2 @@
+<?php
+require_once __DIR__ . '/../../../../../../../../src/lkwdwrd/mu-loader.php';
```

Comment:
not sure these are meant to be committed?

Reply:
ah yeah this shouldn't be

Reply:
Fixed.